docs/source/conf.py

```diff
@@ -69,7 +70,7 @@
 
 # General information about the project.
 project = u'pyresample'
-copyright = u'2013, Esben S. Nielsen'
+copyright = "2013-{datetime.utcnow():%Y}, the Pytroll team"
```

Comment:
Don't you need an `f` to make this an f-string...and only then would stickler be incorrect.

Reply:
Also I think in our license headers on files we've stuck with "pyresample developers". I don't feel strongly either way what this copyright says but maybe the consistency would be nice.

Reply:
I changed my mind, I do prefer the consistency.

```suggestion
copyright = f"2013-{datetime.utcnow():%Y}, Pyresample developers"
```